draft-ietf-taps-transport-security.md

```diff
@@ -1038,8 +1038,18 @@ analysis may compromise any protocol discussed in this survey. Applications usin
 Security Interfaces SHOULD take such limitations into consideration when using a particular
 protocol implementation.
 
+# Privacy Considerations
+
+Privacy enhancing (or offending) features are intentionally omitted from this survey.
```

Comment:
Perhaps, `Analysis of how features improve or degrade privacy is intentionally omitted from this survey.`

Reply:
WFM!